Makefile

```diff
@@ -1,7 +1,7 @@
 SRCPATH		:= $(shell pwd)
 VERSION		:= $(shell $(SRCPATH)/mule/scripts/compute_build_number.sh)
 OS_TYPE		?= $(shell $(SRCPATH)/mule/scripts/ostype.sh)
-ARCH			?= $(shell $(SRCPATH)/mule/scripts/archtype.sh)
+ARCH		?= $(shell $(SRCPATH)/mule/scripts/archtype.sh)
```

Comment:
this should not be changed

Reply:
I have not made these changes. This is coming after syncing the indexer. In my opinion, there will be fewer conflicts if both repos use the same formatting and style.